### PR TITLE
Redeploy Turing cluster to uksouth

### PR DESCRIPTION
**Pull request by callummole**
Actions:

- A new Turing cluster, named `turing-prod`, is deployed to azure region `uksouth` (see https://github.com/jupyterhub/team-compass/issues/529). It has the same specs as the previous one barring increase node capacity and an update k8s version (1.23.8).
- Binderhub deployed successfully (see https://binder.mybinder.turing.ac.uk/)
- Reinstated CI deployment. 
- Reinstated Turing in federation.

### CODE REVIEW

**Comment by callummole** [comment]
do not merge yet - redeploying with updated k8s version

**Comment by callummole** [comment]
Ok redeployed with 1.23.8, ready to merge at your discretion.

**Comment by manics** [comment]
I've hit `Failed to connect to event stream` in every new build lasting more than a few minutes, though refreshing works. Is there a timeout somewhere that causes the event-stream to be cut off?

**Comment by sgibson91** [comment]
@manics I was under the impression that `Failed to connect to event stream` is a browser-side issue (since as you say, refreshing works)?

**Review by sgibson91 — APPROVED** [review]
Suggested comment but LGTM!

**Comment by manics** [comment]
@sgibson91 Not sure.... it's often a browser issue, but when it happens frequently with one server but not another it suggests there may be some other factor.

**Comment by sgibson91** [comment]
Well I don't believe the config for Turing prod has changed at all. It was only torn down and redeployed in a new data centre, so maybe it's something to do with the availability of that data centre? `uksouth`

**Comment by callummole** [comment]
Yes, I haven't changed any timeout values. 

@manics I'm not sure what to look for here, do you have any suggestions for me to investigate?

File: config/turing.yaml [hunk 1]
```diff
@@ -133,7 +129,7 @@ prometheus:
 ingress-nginx:
   controller:
     service:
-      loadBalancerIP: 20.76.176.114
+      loadBalancerIP:
```
Comment (sgibson91) [inline config/turing.yaml:132]: We should either update this field with the new IP address of the ingress-nginx controller, or remove the key completely

Comment (callummole) [reply config/turing.yaml]: updated in [419ccd4](https://github.com/callummole/mybinder.org-deploy/commit/419ccd4b347292f987e45f551409cfe880dddea5)

**Pull request merged**